src/spikeinterface/core/tests/test_recording_tools.py

```diff
@@ -24,14 +24,25 @@ def test_get_closest_channels():
 def test_get_noise_levels():
     rec = generate_recording(num_channels=2, sampling_frequency=1000., durations=[60.])
 
-    noise_levels = get_noise_levels(rec, return_scaled=False)
+    noise_levels_1 = get_noise_levels(rec, return_scaled=False)
+    noise_levels_2 = get_noise_levels(rec, return_scaled=False)
 
     rec.set_channel_gains(0.1)
     rec.set_channel_offsets(0)
-    noise_levels = get_noise_levels(rec, return_scaled=True)
+    noise_levels = get_noise_levels(rec, return_scaled=True, force_recompute=True)
 
     noise_levels = get_noise_levels(rec, return_scaled=True, method="std")
 
+    # Generate a recording following a gaussian distribution to check the result of get_noise.
+    std = 6.0
+    traces = np.random.normal(loc=10.0, scale=std, size=(1_000_000, 2))
```

Comment:
Test that use random numbers should be deterministic.

Reply:
done